can_tools/scrapers/base.py

```diff
@@ -41,6 +47,38 @@ class ValidateDataFailedError(Exception):
     """Error raised when data vailidation fails."""
 
 
+class ValidateRelativeOrderOfCategoriesError(Exception):
+    def __init__(self, cat_small, cat_large, problems):
```

Comment:
FWIW- I have trouble reading this as "category" since "cat" has a very well-defined meaning in my head already... and while it's fun to have large and small cats in the code, it may not lend to readability. 😁 

Furthermore, why are these "categories" and not "columns" or "variables" or something?  Are things like "total_vaccine_initiated" considered a category in the codebase?

Reply:
Good questions.

Happy to spell out category.

And yes, we call a "variable" a triplet of (category, measurement, unit). So something like (total_vaccine_initiated, cumulative, people) is a "variable"

It isn't column because data all long form, so the data are all in the  `value` column